logs/cloudwatch/types.go

```diff
@@ -0,0 +1,54 @@
+package cloudwatch
+
+import (
+	"time"
+
+	"github.com/flanksource/commons/duration"
+)
+
+// +kubebuilder:object:generate=true
+type Request struct {
+	// The log group on which to perform the query.
+	LogGroup string `json:"logGroup" template:"true"`
+
+	// The query to perform on the log group.
+	Query string `json:"query" template:"true"`
+
+	// The maximum number of log events to return in the query.
```

Comment:
The limit, start and end should probably be in a common loqRequest type - and support datemath or timestamp and convert into whatever the upstream needs